manifests/profile/postgres.pp

```diff
@@ -0,0 +1,23 @@
+class puppet_metrics_dashboard::profile::postgres (
+  $pg_password
+){
+
+  pe_postgresql::server::pg_hba_rule{ 'allow telegraf user':
+    type        => 'hostnossl',
+    database    => 'pe-puppetdb',
+    user        => 'telegraf',
+    auth_method => 'password',
+    address     => '192.168.0.8/32',
```

Comment:
Will this be `postgres_host` after the testing phase is done?

Reply:
Oh that makes more sense now. How about a puppetdb_query? We could look for the main class and pull the fact from the node or depend on the certname to be resolvable.